server/controllers/jointController.js

```diff
@@ -0,0 +1,23 @@
+// controller for user middleware
+const db = require('../model/dbModel');
+
+const jointController = {};
+
+jointController.combineUsersAndChores = async (req, res, next) => {
+  try {
+    const chores = res.locals.chores;
+    const users = res.locals.users;
+    res.locals.usersAndChores = {
+      chores: chores,
```

Comment:
Since the property and the variable have the same name you can just say 
res.locals.usersAndChores = {
chores,
users
}
js knows that you want that to be the key and the value.